### PR TITLE
Fix compiler errors when DEBUG is defined

### PR DESCRIPTION
**Pull request by AlexeyKhrabrov**
This PR fixes compiler errors when the compile time flag `DEBUG` is defined.

### CODE REVIEW

File: runtime/compiler/runtime/MethodMetaData.c [hunk 1]
```diff
@@ -333,7 +333,7 @@ static void printMetaData(J9TR_MethodMetaData * methodMetaData)
    for (index = 0; index < iter._stackAtlas->numberOfMaps; ++index)
       {
       getNextMap(&iter, fourByteOffsets);
-      printf("index: %d map address: %p rangeStartOffset: %x rangeEndOffset: %x is inlineMap? %s\n", index, iter._currentMap, iter._rangeStartOffset, iter._rangeEndOffset, IS_BYTECODEINFO_MAP(fourByteOffsets, iter._currentMap) ? "yes" : "no");
+      printf("index: %d map address: %p rangeStartOffset: %zx rangeEndOffset: %zx is inlineMap? %s\n", index, iter._currentMap, iter._rangeStartOffset, iter._rangeEndOffset, IS_BYTECODEINFO_MAP(fourByteOffsets, iter._currentMap) ? "yes" : "no");
```
Comment (mpirvu) [inline runtime/compiler/runtime/MethodMetaData.c:336]: I think that the bullet proof solution for printing uintptr_t is to use `PRIxPTR`. It's ugly, but it's what the standard recommends.

Comment (AlexeyKhrabrov) [reply runtime/compiler/runtime/MethodMetaData.c]: Changed to use `OMR_PRIxPTR` like other places in the JIT.

**Review by mpirvu — APPROVED** [review]
LGTM

**Comment by mpirvu** [comment]
Since the changes are only seen under debug mode, testing will not do much here.

**Pull request merged**